resources/lang/en/people.php

```diff
@@ -10,28 +10,28 @@
 
     //index
     'people_not_found' => 'Contact not found',
-    'people_list_number_kids' => '1 kid|:count kids',
+    'people_list_number_kids' => 'One child|:count children',
     'people_list_last_updated' => 'Last consulted:',
-    'people_list_number_reminders' => '1 reminder|:count reminders',
+    'people_list_number_reminders' => 'One reminder|:count reminders',
     'people_list_blank_title' => 'You don’t have anyone in your account yet',
     'people_list_blank_cta' => 'Add someone',
     'people_list_sort' => 'Sort',
-    'people_list_stats' => '1 contact|:count contacts',
+    'people_list_stats' => 'One contact|:count contacts',
     'people_list_firstnameAZ' => 'Sort by first name A → Z',
     'people_list_firstnameZA' => 'Sort by first name Z → A',
     'people_list_lastnameAZ' => 'Sort by last name A → Z',
     'people_list_lastnameZA' => 'Sort by last name Z → A',
-    'people_list_lastactivitydateNewtoOld' => 'Sort by last activity date newest to oldest',
-    'people_list_lastactivitydateOldtoNew' => 'Sort by last activity date oldest to newest',
+    'people_list_lastactivitydateNewtoOld' => 'Sort by last activity date, newest to oldest',
+    'people_list_lastactivitydateOldtoNew' => 'Sort by last activity date, oldest to newest',
     'people_list_filter_tag' => 'Showing all the contacts tagged with',
     'people_list_clear_filter' => 'Clear filter',
-    'people_list_contacts_per_tags' => '1 contact|:count contacts',
+    'people_list_contacts_per_tags' => 'One contact|:count contacts',
```

Comment:
I don't see any reason not to keep the number even for "1"
```suggestion
    'people_list_contacts_per_tags' => '1 contact|:count contacts',
```